lang/rust/avro/src/types.rs

```diff
@@ -1465,6 +1510,56 @@ Field with name '"b"' is not a member of the map items"#,
         assert!(value.resolve(&Schema::TimestampMicros).is_err());
     }
 
+    #[test]
+    fn resolve_to_nullable_union() {
```

Comment:
please prefix the method name with `avro_1234` where `1234` is the number of the JIRA ticker you create